scripts/assets/maven_dependencies.textproto

```diff
@@ -0,0 +1,875 @@
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "de.hdodenhof:circleimageview:3.0.1"
+  artifact_version: "3.0.1"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "io.fabric.sdk.android:fabric:1.4.7"
+  artifact_version: "1.4.7"
+  license {
+    license_name: "Terms of Service for Firebase Services"
+    original_link: "https://fabric.io/terms"
+    direct_link_only {
+      url: "https://firebase.google.com/terms"
+    }
+    is_original_link_invalid: true
+  }
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://raw.githubusercontent.com/firebase/firebase-android-sdk/master/LICENSE"
+    scrapable_link {
+      url: "https://raw.githubusercontent.com/firebase/firebase-android-sdk/master/LICENSE"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "io.github.chaosleung:pinview:1.4.4"
+  artifact_version: "1.4.4"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "javax.annotation:javax.annotation-api:1.3.2"
+  artifact_version: "1.3.2"
+  license {
+    license_name: "CDDL + GPLv2 with classpath exception"
+    original_link: "https://github.com/javaee/javax.annotation/blob/master/LICENSE"
+    local_copy_link {
+      url: "https://raw.githubusercontent.com/oppia/oppia-android-licenses/develop/javax.annotation%3Ajavax.annotation-api%3A1.3.2-license.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "javax.inject:javax.inject:1"
+  artifact_version: "1"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "net.ltgt.gradle.incap:incap:0.3"
+  artifact_version: "0.3"
+  license {
+    license_name: "The Apache License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "nl.dionsegijn:konfetti:1.2.5"
+  artifact_version: "1.2.5"
+  license {
+    license_name: "Konfetti License"
+    original_link: "https://github.com/DanielMartinus/Konfetti/blob/main/LICENSE"
+    local_copy_link {
+      url: "https://raw.githubusercontent.com/oppia/oppia-android-licenses/develop/nl.dionsegijn%3Akonfetti%3A1.2.5-license.txt"
```

Comment:
Ditto here--prefer pointing directly to the GitHub version since it's available.

Reply:
Done.